### ISSUE TITLE
Fix Stats

### ISSUE THREAD

**Issue by spbyrne** [spbyrne/Murdle#1]
Currently the stats system was designed for the 6 word limit. I'd like to address this and add stats for deaths.

**Comment by coryveilleux** [comment]
See also #3

**Comment by protist** [comment]
Thanks for this great game! However, I just realised that despite the "How to play" saying "You can guess as many words as you want", this doesn't seem to be true? I've played 60 games, and I've only just realised that if you take too many turns to solve it, even if you don't "die", this still seems to count as a fail? Presumably you *do* only have six turns to solve the puzzle?

**Comment by coryveilleux** [comment]
@protist, for me at least the share emoji doesn't indicate a loss... I just took 8 tries to complete today's murdle while only having 2 deaths out of the 10:

:skull:MURDLE #381 - 8
:green_square::green_square::green_square::green_square::green_square:
:yellow_square::yellow_square::black_large_square::yellow_square::black_large_square:
:yellow_square::yellow_square::black_large_square::black_large_square::yellow_square:
:yellow_square::yellow_square::black_large_square::black_large_square::yellow_square:
:yellow_square::yellow_square::black_large_square::yellow_square::black_large_square:
:green_square::black_large_square::green_square::green_square::black_large_square:
:green_square::black_large_square::yellow_square::black_large_square::yellow_square:
:green_square::black_large_square::black_large_square::yellow_square::yellow_square:

But based on my overall stats, nothing larger than 6 shows up in the list. Is that what you mean? I'm not sure if the success rate only includes 6 and under...

![image](https://user-images.githubusercontent.com/757322/212940491-99cb2c55-2f6c-4d9d-a1ea-2141c2b8cff1.png)

**Comment by protist** [comment]
@coryveilleux Yes, your last point was what I meant. The "Success rate" appears to be based on "wins" that take ≤6 turns. Record the success rate before a game, solve it in >6 turns, then check again. (I just tried to do this but failed instead 😵‍💫)

**Comment by protist** [comment]
Okay, I completed a game in nine turns, with three hearts left.

💀MURDLE #383 - 9

🟩🟩🟩🟩🟩
⬛🟩🟩🟩🟩
⬛🟨🟩⬛⬛
🟩🟩⬛⬛⬛
🟨🟨🟨🟩⬛
🟨🟨⬛🟩⬛
🟨🟨⬛⬛⬛
🟨⬛⬛⬛⬛
⬛⬛⬛🟩⬛

77% success before
<img src="https://user-images.githubusercontent.com/646668/213180986-27fac5af-c737-4be3-97d3-ea3c93074134.png" width=20% height=20%>

And it drops to 76% success after
<img src="https://user-images.githubusercontent.com/646668/213181581-81b83c2a-2c6f-4ea1-a3aa-468ccee7e1b3.png" width=20% height=20%>
